security-release-process-documentation/security-release-process.md

```diff
@@ -114,6 +149,16 @@ Who do you call when the cluster is not well or you gotta test something weird,
 
 Handles public messaging around the bug. Documentation on how to upgrade. Changelog. Explaining to public the severity. Notifications of bugs sent to mailing lists etc. Requests CVEs.
 
+##### Associate
+
+A role for those wishing to join the PST. They should not currently be a member
+of security@kubernetes.io. 
+
+Their rotation will involve the following:
+
+- leading disclosures pushed to publicly by PST due to low CVSS score.
```

Comment:
not sure i understand this sentence. does this mean?

`- publish public disclosures when the they have a low CVSS score`?

Reply:
"lead disclosures that are publicly disclosed on a public GitHub issue tracker (often done because of reporter request, a low CVSS score, or design issue that requires long-term refactoring)"

Does that help @dims

Reply:
prefect! thanks @philips

Reply:
thanks you both! updated :)